--- src/src/stdgpu/CMakeLists.txt
+++ src/src/stdgpu/CMakeLists.txt
@@ -4,10 +4,14 @@
 set(STDGPU_BUILD_CMAKE_DIR "${CMAKE_CURRENT_BINARY_DIR}/cmake")
 
 # Setup dependencies
-find_package(thrust 1.9.9 REQUIRED MODULE)
+find_package(Thrust 2.0 REQUIRED CONFIG)
+thrust_create_target(thrust::thrust)
 
 set(STDGPU_DEPENDENCIES_INIT "
-find_dependency(thrust 1.9.9 REQUIRED MODULE)
+if(NOT TARGET thrust::thrust)
+    find_package(Thrust 2.0 REQUIRED CONFIG)
+    thrust_create_target(thrust::thrust)
+endif()
 ")
 
 
@@ -181,10 +185,6 @@
         DESTINATION "${STDGPU_CMAKE_INSTALL_DIR}"
         COMPONENT stdgpu)
 
-install(FILES "${stdgpu_SOURCE_DIR}/cmake/Findthrust.cmake"
-        DESTINATION "${STDGPU_CMAKE_INSTALL_DIR}"
-        COMPONENT stdgpu)
-
 install(FILES "${stdgpu_SOURCE_DIR}/cmake/${STDGPU_BACKEND_DIRECTORY}/determine_thrust_paths.cmake"
         DESTINATION "${STDGPU_CMAKE_INSTALL_DIR}/${STDGPU_BACKEND_DIRECTORY}"
         COMPONENT stdgpu)
