--- src/cub/cmake/CubInstallRules.cmake
+++ src/cub/cmake/CubInstallRules.cmake
@@ -11,12 +11,12 @@
 )
 
 install(DIRECTORY "${CUB_SOURCE_DIR}/cub/cmake/"
-  DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/cub"
+  DESTINATION "share/cub"
   REGEX .*header-search.cmake.* EXCLUDE
 )
 # Need to configure a file to store the infix specified in
 # CMAKE_INSTALL_INCLUDEDIR since it can be defined by the user
-set(install_location "${CMAKE_INSTALL_LIBDIR}/cmake/cub")
+set(install_location "share/cub")
 configure_file("${CUB_SOURCE_DIR}/cub/cmake/cub-header-search.cmake.in"
   "${CUB_BINARY_DIR}/cub/cmake/cub-header-search.cmake"
   @ONLY)
